e2e-tests/insurance/cypress/e2e/journeys/export-contract/about-goods-or-services/about-goods-or-services.spec.js

```diff
@@ -170,7 +170,7 @@ context('Insurance - Export contract - About goods or services page - Final dest
         });
 
         it('allows user to remove a selected country and search again', () => {
-          checkAutocompleteInput.allowsUserToRemoveCountryAndSearchAgain(field, COUNTRIES[0].NAME, COUNTRIES[1].NAME);
+          checkAutocompleteInput.allowsUserToRemoveCountryAndSearchAgain(field, COUNTRIES[0].NAME, COUNTRIES[1].NAME, COUNTRIES[1].NAME);
```

Comment:
Could we update the E2E countries fixtures to export individal countries? Then this can be easier to read and no need to hard code country names in the above test - thinking e.g

```
const XAD = { ... }
const DZA = { ... }
```

Have just done the same for currencies